apps/test/levelTests.js

```diff
@@ -70,6 +70,8 @@ describe('Level tests', function() {
     if (window.Studio) {
       var Projectile = require('@cdo/apps/studio/projectile');
       Projectile.__resetIds();
+      var Item = require('@cdo/apps/studio/Item');
```

Comment:
Unrelated to this PR, but seemed like a good idea to __resetIds() for Items since it has the same function. Once we start writing Item tests, we'll be glad we did this.